device/arista/x86_64-arista_7060_cx32s/Arista-7060CX-32S-Q24C8/qos.json.j2

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
{%- include 'qos_config.j2' %}

check other skus, it is simplified.

Reply:
Agreed, I have corrected the file